keybindings.js

```diff
@@ -1,51 +1,47 @@
-var Extension;
-if (imports.misc.extensionUtils.extensions) {
-    Extension = imports.misc.extensionUtils.extensions["paperwm@hedning:matrix.org"];
-} else {
-    Extension = imports.ui.main.extensionManager.lookup("paperwm@hedning:matrix.org");
-}
+var ExtensionUtils = imports.misc.extensionUtils;
+var Extension = ExtensionUtils.getCurrentExtension();
+var ExtensionModule = Extension.imports.extension;
 
-var Me = Extension.imports.keybindings;
-var Gdk = imports.gi.Gdk;
-var Gtk = imports.gi.Gtk;
-var Gio = imports.gi.Gio;
+var Clutter = imports.gi.Clutter;
+var Seat = Clutter.get_default_backend().get_default_seat();
 var Meta = imports.gi.Meta;
-var GLib = imports.gi.GLib;
 
 var Utils = Extension.imports.utils;
 var Main = imports.ui.main;
 var Shell = imports.gi.Shell;
 
-var convenience = Extension.imports.convenience;
 var Settings = Extension.imports.settings;
 var keystrToKeycombo = Settings.keystrToKeycombo;
 
 var Navigator = Extension.imports.navigator;
 var Tiling = Extension.imports.tiling;
+var LiveAltTab = Extension.imports.liveAltTab;
+var Scratch = Extension.imports.scratch;
+var App = Extension.imports.app;
 
 var display = global.display;
 
-var KEYBINDINGS_KEY = 'org.gnome.Shell.Extensions.PaperWM.Keybindings';
+var KEYBINDINGS_KEY = 'org.gnome.shell.extensions.paperwm.keybindings';
```

Comment:
Can we actually use this constant string in the functions below?

Reply:
Oh good point - yeah, we should actually use in functions!